examples/sha256-hashes/sha256-hashes.go

```diff
@@ -1,26 +1,25 @@
-// [_SHA1 hashes_](http://en.wikipedia.org/wiki/SHA-1) are
+// [_SHA256 hashes_](https://en.wikipedia.org/wiki/SHA-2) are
 // frequently used to compute short identities for binary
-// or text blobs. For example, the [git revision control
-// system](http://git-scm.com/) uses SHA1s extensively to
-// identify versioned files and directories. Here's how to
-// compute SHA1 hashes in Go.
+// or text blobs. For example, TLS/SSL certificates use SHA256
+// to compute a certificate's signature. Here's how to compute
+// SHA256 hashes in Go.
 
 package main
 
 // Go implements several hash functions in various
 // `crypto/*` packages.
 import (
-	"crypto/sha1"
+	"crypto/sha256"
 	"fmt"
 )
 
 func main() {
-	s := "sha1 this string"
+	s := "sha256 this string"
 
-	// The pattern for generating a hash is `sha1.New()`,
-	// `sha1.Write(bytes)`, then `sha1.Sum([]byte{})`.
+	// The pattern for generating a hash is `sha256.New()`,
```

Comment:
Can you remove this comment entirely while you're at it? It seems wrong (the `Sum` call doesn't match the docs or the code) too

Reply:
Removed the comment.